### PR TITLE
error log clearer

### PR DESCRIPTION
**Pull request by Yang09701194**
See #720

### CODE REVIEW

**Comment by SyntevoAlex** [comment]
With tabs fixed, it can be merged next week when code freeze period is over.

**Comment by iloveeclipse** [comment]
> With tabs fixed, it can be merged next week when code freeze period is over.

Please provide **meaningful** commit message before merge.
See https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md#commit-message-recommendations

**Comment by SyntevoAlex** [comment]
Ah right, I wanted to ask to change it, but then forgot. Thanks for a reminder.
I suggest the following message
`[win32] Natives build: Easier to understand error when Visual Studio is not installed`

**Comment by iloveeclipse** [comment]
> I suggest the following message `[win32] Natives build: Easier to understand error when Visual Studio is not installed`

I suggest to have slightly improved variant:
```
[win32] Natives build: Easier to understand error when Visual Studio is not installed

Fixes https://github.com/eclipse-platform/eclipse.platform.swt/issues/720
```
... which would 
- fix linked issue #720 automatically after merge
- and allow anyone to jump to the issue from git history with one click

File: bundles/org.eclipse.swt/Eclipse SWT PI/win32/library/build.bat [hunk 1]
```diff
@@ -70,12 +70,17 @@ shift
 @rem if call to vcvarsall.bat (which sets up environment) silently fails, then provide advice to user.
 WHERE cl
 if %ERRORLEVEL% NEQ 0 (
-    CALL :ECHO "WARNING: cl (Microsoft C compiler) not found on path. Please install Microsoft Visual Studio."
+    CALL :ECHO "ERROR: cl (Microsoft C compiler) not found on path. Please install Microsoft Visual Studio."
     CALL :ECHO "         If already installed, try launching eclipse from the 'Developer Command Prompt for VS'"
     CALL :ECHO "         Refer steps for SWT Windows native setup: https://www.eclipse.org/swt/swt_win_native.php"
+	CALL :ECHO "ERROR: exit 1"
```
[the remark this answers is not in the document]
Comment (Yang09701194) [reply bundles/org.eclipse.swt/Eclipse SWT PI/win32/library/build.bat]: have change tabs to spaces use git push force

**Comment by Yang09701194** [comment]
have modifed message use `git push force`

**Comment by SyntevoAlex** [comment]
In master now.
Thanks for your contribution!

**Pull request merged**